pod/completion/models.py

```diff
@@ -139,7 +139,7 @@ def __str__(self):
 
     def get_base_mail(self):
         data = base64.b64encode(self.email_address.encode())
```

Comment:
Je propose d'ajouter cette pydoc :
"""Encode an email as base64 to prevent robots crawling."""

Reply:
et tant qu'on y est, ajouter `-> str` au type de la fonction ;)

Reply:
fait